src/pages/instances/InstanceOverviewNetworks.tsx

```diff
@@ -64,7 +64,7 @@ const InstanceOverviewNetworks: FC<Props> = ({ instance, onFailure }) => {
             content: (
               <Link
                 to={`/ui/project/${instance.project}/networks/detail/${network.name}`}
-                title={network.name}
+                title={`link to ${network.name}`}
```

Comment:
I think we should avoid putting semantics like "this is a link" into captions. Could say "Network foo" though.